datalad/__init__.py

```diff
@@ -9,21 +9,17 @@
 """DataLad aims to expose (scientific) data available online as a unified data
 distribution with the convenience of git-annex repositories as a backend."""
 
-from .log import lgr
 
 # Other imports are interspersed with lgr.debug to ease troubleshooting startup
 # delays etc.
-lgr.log(5, "Instantiating config")
 from .config import ConfigManager
 cfg = ConfigManager()
 
-lgr.log(5, "Instantiating ssh manager")
```

Comment:
couldn't we keep this one already since config is already imported and now we could import log?

asking because all those log lines were important/used to optimize import times, so if we are to prune -- we should prune as little as sufficient